File: leetcode-04/solutions/python3/0448.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        ans = []

        for num in nums:
            nums[abs(num) - 1] = -abs(nums[abs(num) - 1])

        for i in range(len(nums)):
            if nums[i] > 0:
                ans.append(i + 1)

        return ans

File: leetcode-04/solutions/cpp/0448.cpp
class Solution {
 public:
  vector<int> findDisappearedNumbers(vector<int>& nums) {
    vector<int> ans;

    for (int num : nums) nums[abs(num) - 1] = -abs(nums[abs(num) - 1]);

    for (int i = 0; i < nums.size(); i++)
      if (nums[i] > 0) ans.push_back(i + 1);

    return ans;
  }
};